Gemfile

```diff
@@ -6,6 +6,9 @@ ruby "~> #{File.read(File.join(__dir__, '.ruby-version')).strip}"
 gem 'rails', '~> 7.1.3'
 
 gem 'ahoy_matey', '~> 3.0'
+# pod identity requires 3.188.0
+# https://docs.aws.amazon.com/eks/latest/userguide/pod-id-minimum-sdk.html
```

Comment:
I'd consider dropping the comment since if we did a `git blame` we'd find the PR wit the comment (and commit message)

Reply:
i like the comments because it saves a step from having to git blame to find out a why, it's right there in the code

Reply:
fair enough! I think if every gem we had to bump had a comment like that the file would be noisy, but for a single one like this, it's probably ine